src/prognostic_equations/advection.jl

```diff
@@ -424,7 +428,7 @@ function edmfx_sgs_vertical_advection_tendency!(
                     FT(1),
                     Y.c.sgsʲs.:($$j).ρa,
                     FT(0),
-                    Y.c.ρ,
+                    ᶜρʲs.:($$j),
```

Comment:
Is this correct? I thought we should tend to grid mean density if area is small?